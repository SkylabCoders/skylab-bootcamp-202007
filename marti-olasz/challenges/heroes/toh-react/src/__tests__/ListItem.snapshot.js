import renderer from 'react-test-renderer';
import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import ListItem from '../components/HeroListItem';

describe('ListItems snapshot', () => {
	const tree = renderer.create(
		<Router>
			<ListItem />
		</Router>
	);
	it('Should match', () => {
		expect(tree.toJSON()).toMatchSnapshot();
	});
});
