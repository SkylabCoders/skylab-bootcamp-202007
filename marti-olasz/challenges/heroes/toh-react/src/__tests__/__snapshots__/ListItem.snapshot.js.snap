// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ListItems snapshot Should match 1`] = `
<p>
  HeroListItem works!
</p>
`;
